### ISSUE TITLE
?

### ISSUE THREAD

**Issue by elvis13-prog** [iceland2k14/kangaroo#1]
The program does not find private keys, for 60 hex. What could be the reason? In general, he does not find any !!!

**Comment by iceland2k14** [comment]
Change this line to 0. If you are testing and sure about that key is 100% inside the narrow range.
https://github.com/iceland2k14/kangaroo/blob/59fb5d5a9a07e3ef320e1b83630d5cfe25e011e5/kangaroo.py#L91
Otherwise we keep value as 2 or 3, when we don't know anything.

**Comment by elvis13-prog** [comment]
I correctly understood that if you do not put keyspace , then you can search for any pub key? With an unknown range?

**Comment by iceland2k14** [comment]
Yes if you don't give keyspace it will take the Full range as Default. 1:FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364140
